team-telnyx/webrtc
# feat(WEBRTC-317): add methods to check webrtc compatibility and list the supported browser

- add a new method to check if the current browser has supported to use TelnyxRTC
- add a new method to list supported system and browser by TelnyxRTC
- add type docs for the above methods
- add unit test for the above methods

## 📝 To Do


## ✋ Manual testing

1. Navigate into `packages/js` and run `npm run build`
2. Navigate into `js/examples/react-audio` and run `npm setup`
3. Access http://localhost:6006/?path=/story/webdialer--example and open the inspect browser
4. Open console and type this `TelnyxRTC.webRTCInfo()` to know if the browser supports TelnyxRTC
5. Open console and type this `TelnyxRTC.webRTCSupportedBrowserList()` to know the supported browsers
6. You should test in different browsers and see the results.
7.  Navigate into `packages/js` and `npm run test` you should get all green.
8. Navigate into `packages/js` and `npm run docs` you should see the new methods documented on `TelnyxRTC.MD`

## 🦊 Browser testing

### Desktop


## 📸 Screenshots

### Supported browser result
![image](https://user-images.githubusercontent.com/16343871/105089531-05855200-5a7c-11eb-8893-56c5c5a9d3a4.png)

### *Not* supported browser
![image](https://user-images.githubusercontent.com/16343871/105089615-1d5cd600-5a7c-11eb-9e59-e6eeaa978e34.png)
